### PR TITLE
chore/adds-navbar

### PR DESCRIPTION
**Pull request by wangui-wamutitu**
Adds navbar section

Screenshot 2024-08-21 at 00 47 53

### CODE REVIEW

**Review by cindykandie — CHANGES_REQUESTED** [review]
I really like the rendercon logo addition that quite brilliant.

File: src/app/components/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Link from 'next/link'
+import React from 'react'
+
+function Navbar() {
+  return (
+    <ul className="w-[400px] md:w-[500px] lg:w-[600px] flex font-mono justify-between items-center">
+        <li><Link href="#">Agenda</Link></li>
+        <li><Link href="#speakers">Speakers</Link></li>
```
[the remark this answers is not in the document]
Comment (wangui-wamutitu) [reply src/app/components/Navbar.tsx:8]: Is there an about section/page? I can't see one

File: src/app/components/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        <li><Link href="#sponsors">Sponsors</Link></li>
+        <li><Link href="#contacts">Contacts</Link></li>
+    </ul>
+  )
+}
+
```
[the remark this answers is not in the document]
Comment (wangui-wamutitu) [reply src/app/components/Navbar.tsx:14]: On it

[the remark this answers is not in the document]
Comment (wangui-wamutitu) [reply src/app/components/Navbar.tsx:14]: @cindykandie, responsiveness for small screens done

Screenshot 2024-08-22 at 12 56 48
Screenshot 2024-08-22 at 12 56 57

File: src/app/components/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import Link from 'next/link'
+import React from 'react'
+
+function Navbar() {
+  return (
+    <ul className="w-full lg:w-[600px] flex flex-col h-[200px] lg:h-auto lg:flex-row font-mono justify-between lg:items-center">
+        <li><Link href="#">Agenda</Link></li>
```
Comment (cindykandie) [inline src/app/components/Navbar.tsx:7]: hey, let's comment out/leave out about and agenda so we can add them after the pages are created.

**Review by cindykandie — CHANGES_REQUESTED** [review]
Hey Tish, let's comment out/leave out about and agenda navigators so we can add them after the pages are created.

You can rebase with the main branch and re-request review for merging

**Review by cindykandie — APPROVED** [review]

**Pull request merged**